lib/signaling/v2/twilioconnectiontransport.js

```diff
@@ -323,6 +327,28 @@ class TwilioConnectionTransport extends StateMachine {
     return false;
   }
 
+  resumeConnect() {
+    this._sendConnectOrSyncOrDisconnectMessage();
+  }
+
+  _sendIceMessage() {
+    this._twilioConnection.sendMessage({
+      edge: this._options.region ? this._options.region : 'roaming',
+      token: this._accessToken,
+      type: 'ice',
+      version: 1
+    });
+  }
+
+  connect() {
```

Comment:
@makarandp0 Since this is called only by this class and not outside, we should make this a private method like `_startConnect()`.